use-timescale/integrations/query-admin/pgadmin.md

```diff
@@ -15,4 +15,92 @@ various Linux flavors.
+
+<Tabs label="Connect to Timescale with pgAdmin">
+
+<Tab title="Timescale">
+
+<Procedure>
+
+1.  Sign in to the [Timescale portal][tsc-portal].
+1.  In the `Services` tab, find the service you want to connect to, and check
+    it is marked as `Running`.
+1.  Click the name of the service you want to connect to, and take a note
+     of the `Host`, `Port`, `Database name`, and `Username`.
+1.  Navigate to the `Operations` tab, and click `Reset password`. You can choose
+    your own password for the service, or allow Timescale to generate a
+    secure password for you. Take a note of your new password.
+1.  Start `pgAdmin`.
+1.  In the `Quick Links` section of the `Dashboard` tab, click `Add New Server`.
+1.  In the `Register - Server` page go to the `General` tab.
+1.  In the `Name` field, type a name for the server, and the description in the
+    `Comments` field.
+1.  In the `Connection` tab, fill in the fields using the server name, database,
+    user name, port, and password for your Timescale service. For more
+    information about these settings,
+    see [the connection section][connection-details].
+1.  Click `Save` to save the configuratiosn that you set.
+
+    <Highlight type="note">
+    If you configured your Timescale service to connect using [SSL
+    mode](https://docs.timescale.com/use-timescale/latest/security/strict-ssl/),
+    in the `Parameters` tab, set `SSL mode` to `Verify-Full`, and set the
+    location of the SSL root CA certificate to use.
+    </Highlight>
+
+1.  Click `Save`.
+
+    <img class="main-content__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/pgadmin_timescale_cloud.webp"
+    alt="Connecting to Timescale using pgAdmin"/>
+
+</Procedure>
+
+</Tab>
+
+<Tab title="Self-hosted Timescale">
+
+<Procedure>
+
+1.  Start `pgAdmin`.
+1.  In the `Quick Links` section of the `Dashboard` tab, click `Add New Server`.
+1.  In the `Register - Server` page go to the `General` tab.
+1.  In the `Name` field, type a name for the server, and the description in the
+    `Comments` field.
+1.  In the `Connection` tab, fill in the fields using the server name, database,
+    user name, port, and password for your Timescale service. For more
+    information about these settings,
+    see [the connection section][connection-details].
+1.  Click `Save` to save the configuratiosn that you set.
+
+    <img class="maincontent__illustration"
+    src="https://s3.amazonaws.com/assets.timescale.com/docs/images/pgadmin_self_hosted.webp"
+    alt="Connecting to Timescale using pgAdmin"/>
+
+</Procedure>
+
+</Tab>
+
+</Tabs>
+
+## Settings in the `Connection`
+
+This table provides the description and example values for the fields in the
+`Connection` dialog.
+
+|Setting|Example value|Description|
+|-|-|-|
+|Host name/address|localhost, postgresql.example.com, <REMOTE_HOST>.tsdb.cloud.timescale.com|The fully qualified server name. The host name of your Timescale service.|
```

Comment:
Maybe markup the example values? Also, the descriptions read a bit awkwardly, can we reword?